ui/src/core/search_manager.ts

```diff
@@ -1,31 +1,17 @@
-// Copyright (C) 2024 The Android Open Source Project
```

Comment:
You probable didn't want to remove this?

Reply:
Good catch. Reverted.